docs/tools/offline/cli-setup.md

```diff
@@ -3,6 +3,8 @@
 
 You can install Mbed CLI on Windows, Linux and Mac OS X.
 
+<span class="tips">If you're working on Windows, we recommend you use the [Windows installer for Mbed CLI](LINK TBC). For Linux and Mac OS X, refer to the section [Installing Mbed CLI standalone stable version](LINK TBC)</span>
```

Comment:
It's weird to me that this is before requirements.

Reply:
Do users really need to know the requirements for Mbed CLI? (python, git, etc?)

I'm concerned about them being overwhelmed by so many requirements, which are resolved by using a single Windows installer package. The only requirement for the Windows installer is having Windows 7, 8 or 10.

Reply:
I would say users installing Mbed CLI need to know any requirements or prerequisites to using it, yes.

Reply:
Amanda, IMO people that want to download the Windows Installer for Mbed CLI to get started with Mbed don't need to worry about too many details - for example, if you want to download Skype, you only care about getting the compatible version for your Windows PC, not about the libraries/apps that get installed as part of that (although it's ok to mention in release notes, etc).

The key section **Install Mbed CLI for Windows installer** containing a link to the .exe is buried in the middle of the doc and the link is not clearly visible. This is not great for new users.
I believe we need either a new section (at the same level as Setup), or bring the topic earlier with a banner as I'm suggesting. We're running many workshops and need to point people to the page for the Windows installer - again not great to have too much info for a new starter.

Reply:
OK. That makes sense to me. Thanks for the comparison. That helps.